src/main/java/essenmakanan/ui/Ui.java

```diff
@@ -30,47 +28,35 @@ public void functionSelect() {
         System.out.println(DIVIDER);
     }
 
-    public void drawDivider() {
+    public static void drawDivider() {
         System.out.println(DIVIDER);
     }
```

Comment:
i like how you change all the methods to static to prevent unnecessary creation of Ui object